lunches.py

```diff
@@ -552,6 +552,26 @@ def assen(dom):
     yield from menicka_parser(dom)
 
 
+@restaurant("Bistro Paulus", "https://www.bistro-paulus.cz/poledni-menu/", Location.Olomouc)
+def paulus(dom):
+    current_day = datetime.datetime.now().strftime("%-d.%-m.%Y")
+    for day_dom in dom.css(".section-day"):
+        day = "".join(day_dom.css_first("h3").text(strip=True).split()[1:])
+        if current_day not in day:
+            continue
+        soup_table = day_dom.css("table")[0].css("span")
+        lunch_table = day_dom.css("table")[1].css("span") + day_dom.css("table")[2].css("span")
+        for i in range(0, len(soup_table), 2):
```

Comment:
what about this? zip even rows with odd rows (`[start:end:increment]`) into tuple:
```python
for soup, price in zip(soup_table[::2], soup_table[1::2]):
            yield Soup(soup.text(strip=True), price.text(strip=True))
```

Reply:
fixde